pylabnet/gui/pyqt/stylesheets/pylabnet copy.css

```diff
@@ -0,0 +1,82 @@
+/* QPushButton{
```

Comment:
do we need this file "pylabnet copy"?

Reply:
Nope, we don't even need the `pylabnet.css`, deleted both.